CMakeLists.txt

```diff
@@ -21,7 +21,6 @@ set(DIR_3RDPARTY ${FileIO_SOURCE_DIR}/3rdparty)
 MESSAGE(" PROJECT_SRC = : ${PROJECT_SRC}")
 include_directories(${PROJECT_SRC})
 SET(ACTIVE_CPP0xx_DIR "Release")
-find_library(TCMALLOC tcmalloc PATHS /usr/local/probe/lib )
```

Comment:
@john-gress :  I don't think it is needed for FileIO to be tcmalloc linked. If you disagree then i could make it so that the tcmalloc library is passed in through a cmake argument.